ahnlich/ai/src/tests/aiproxy_test.rs

```diff
@@ -623,6 +624,179 @@ async fn test_ai_proxy_get_sim_n_succeeds() {
     );
 }
 
+#[tokio::test]
+async fn test_convert_store_input_to_embeddings() {
```

Comment:
Nice idea on the tests... I like it

We can probably parameterize the models with with [test_case](https://docs.rs/test-case/latest/test_case/) crate as a dev dependency for ai

Reply:
Would then mean we can remove the for loop